components/emoji_picker/emoji_picker.test.jsx

```diff
@@ -20,4 +39,119 @@ describe('components/emoji_picker/EmojiPicker', () => {
             expect(filterEmojiSearchInput(testCase.input)).toEqual(testCase.output);
         });
     });
+
+    const actions = {
+        getCustomEmojis: jest.fn(),
+        incrementEmojiPickerPage: jest.fn(),
+        searchCustomEmojis: jest.fn(),
+    };
+
+    const customEmojis = new Map();
+    customEmojis.set('smiley', {name: 'smiley'});
+
+    const emojiMap = new Set();
+    emojiMap.add({customEmojis});
+    emojiMap.add({customEmojisArray: ['smiley']});
+
+    const smiley = new Map();
+    smiley.set('smiley', {fileName: '1f603', category: 'people', aliases: ['smiley']});
+
+    emojiMap.add(smiley);
+
+    const baseProps = {
+        customEmojiPage: 1,
+        customEmojisEnabled: true,
+        listHeight: 245,
+        onEmojiClick: jest.fn(),
+        onEmojiClose: jest.fn(),
+        emojiMap: new EmojiMap([]),
+        recentEmojis: [],
+        actions,
+    };
+
+    test('init snapshot', () => {
+        const wrapper = shallow(
+            <EmojiPicker {...baseProps}/>
+        );
+
+        wrapper.instance().emojiPickerContainer = {
+            offsetHeight: 200,
+        };
+
+        expect(wrapper).toMatchSnapshot();
```

Comment:
i don't think this snapshot adds much value and probably just noise. Just added incase. Let me know other feel the same.

Reply:
I don't think it does not hurt to have it but it doesn't do much